src/main/java/com/faforever/client/chat/event/FocusChannelTabEvent.java

```diff
@@ -0,0 +1,10 @@
+package com.faforever.client.chat.event;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Getter
```

Comment:
Use @Data

Reply:
Does the same

Reply:
No it doesn't, it creates @Setter, @EqualsAndHashcode aswell.